odoo_tools/cli/submodule.py

```diff
@@ -33,7 +34,7 @@ def init():
     ui.echo("")
     ui.echo("You can now update odoo/Dockerfile with this addons-path:")
     ui.echo("")
-    ls()
+    ctx.invoke(ls)
```

Comment:
this caused the CI to fail

Apparently we can't just call a click command directly, we must invoke it